### DISCUSSION TITLE
Installing kalifs-arm64-full.tar.xz using termux

### DISCUSSION THREAD

**Discussion by Sabelo-Wayne** [termux/termux-packages#11996] [Q&A] [answered]
Hello there

I'd like to know how do I install **kalifs-arm64-full.tar.xz** using termux on my android device.

Kind regards,
Wayne

**Comment by OrdinaryEnder** [comment]
What purpose your gonna do with kali nethunter?

**Comment by sylirre** [comment, marked as answer]
Installation is as simple as unpacking this tar archive. Your device should have AArch64 (ARM64) CPU.

**Reply by OrdinaryEnder** [reply]
Since axel can't access https://kali.download, i have patch something (changing download manager)
```bash
curl -LO https://gist.github.com/OrdinaryEnder/b397bf44557018fbe44f12224482aff4/raw/4aec5a61087039e2b918bbebd9d35ae7d4f1a714/kali.sh && bash kali.sh
```
